pkg/cmd/scan/output/assets/index.tmpl

```diff
@@ -119,15 +126,19 @@
                         <div role="rowgroup">
                             <div role="row" class="table-header">
                                 <span role="columnheader">Resource ID</span>
-                                <span role="columnheader">Resource Type</span>
+                                <span role="columnheader">IaC source</span>
                             </div>
                         </div>
                         <div role="rowgroup" class="table-body">
                             {{range $diff := .Differences}}
                             <div role="row" data-kind="resource-changed" class="resource-item">
                                 <div class="row">
-                                    <span role="cell" data-type="resource-id">{{$diff.Res.TerraformId}}</span>
-                                    <span role="cell" data-type="resource-type">{{$diff.Res.TerraformType}}</span>
+                                    <span role="cell">
+                                        <span data-type="resource-id">{{$diff.Res.TerraformId}}</span>
+                                        {{ if $diff.Res.Src }}(<span>{{$diff.Res.SourceString}}</span>){{ end }}
```

Comment:
Miss the `TerraformType` if there is no source I think

Reply:
I think we always have iac source for drifts? but I can add a check anyway